app/models/form_submission_attempt.rb

```diff
@@ -25,15 +25,31 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        user_uuid = form_submission.user_account.id
+        form_type = form_submission&.form_type
+        log_info = { form_submission_id:, benefits_intake_uuid: form_submission&.benefits_intake_uuid, form_type:,
+                     user_uuid: user_uuid }
+        zsf_function = "#{form_id} form submission to Lighthouse"
```

Comment:
Where is `form_id` assigned?

Reply:
Good catch, this should be `form_type`, which is part of the model. Addressed in fee21cb1daf438aec1afb69be9ceecce9b49021c